build.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+set -e 	# Exit immediately upon failure
+set -o pipefail  # Carry failures over pipes
+
+repo_root="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+docker_repo="microsoft/aspnetcore"
```

Comment:
Because this doesn't actually push to dockerhub it doesn't matter. You just need some name to refer to the images that are built to test them.